import java.util.Scanner;

public class TryAndCatchExample
{
	public static void main(String[] args) {
		Scanner s=new Scanner(System.in);
		System.out.println("Enter 2 no's:");
		int a=s.nextInt();
		int b=s.nextInt();
		try
		{
			int c=a/b;//----Create object of ArithmeticException--throw
			System.out.println("Division is :"+c);
		}
		catch (ArithmeticException e)
		{
			System.out.println(e);//toString()
			System.out.println("Can't devide ob zero");
		}
		System.out.println("Rest of the code.....");
	}

}
